### PR TITLE
Allow client to return raw response, if requested.

### PR DESCRIPTION
**Pull request by nathanbrauer**
This allows for thin clients to easily act as proxies (e.g. to access the API securely via Javascript)

@dchesterton - Thank you for building this for Marketo. Please let me know if you have any questions. We plan on using your client in the next version of Marketo.com. :)

ps: creating a new request which pulls from a feature branch.
cc: @MurtzaM

### CODE REVIEW

File: src/Client.php [hunk 1]
```diff
@@ -490,6 +490,8 @@ private function getResult($command, $args, $fixArgs = false)
 
         $cmd->prepare();
 
+		if ($return_raw) return $cmd->getResponse()->getBody(true);
```
Comment (dchesterton) [inline src/Client.php]: If statements should have brackets, and indenting needs fixing.

File: src/Client.php [hunk 2]
```diff
@@ -93,7 +93,7 @@ public static function factory($config = array())
      *
      * @return CreateOrUpdateLeadsResponse
      */
-    private function createOrUpdateLeadsCommand($action, $leads, $lookupField, $args)
+    private function createOrUpdateLeadsCommand($action, $leads, $lookupField, $args, $return_raw = false)
```
Comment (dchesterton) [inline src/Client.php]: $returnRaw instead of $return_raw, needs fixing on all methods

**Comment by dchesterton** [comment]
Thanks Nathan, that's great to hear it's going to be used on Marketo.com :)

PR looks good, I've just left a couple of comments on a couple of formatting/PSR-2 issues. :+1:

**Comment by nathanbrauer** [comment]
Done and ready! :)

**Comment by nathanbrauer** [comment]
@dchesterton - I've pulled in upstream changes so this pull request can now be automatically merged. :)

**Comment by nathanbrauer** [comment]
Hi @dchesterton - sorry to press. Any chance of having this merged soon? I'd like to kill my fork and set composer to reference your repo instead of ours.

**Comment by dchesterton** [comment]
Sorry for the delay with this, all merged now.

**Comment by nathanbrauer** [comment]
Woo hoo! Thanks!

**Pull request merged**